themes/bootstrap3/templates/myresearch/odmycontent.phtml

```diff
@@ -0,0 +1,152 @@
+<?php
+  // Set up page title:
+  $this->headTitle($this->translate('od_content'));
+
+  // Set up breadcrumbs:
+  $this->layout()->breadcrumbs = '<li><a href="' . $this->url('myresearch-home') . '">' . $this->transEsc('Your Account') . '</a></li> <li class="active">' . $this->transEsc('od_content') . '</li>';
+?>
+
+<div class="<?php echo $this->layoutClass('mainbody')?>">
```

Comment:
Just an observation: instead of `<?php echo ... ?>` you can use `<?= ... ?>` to save a bit of typing and improve readability.

Reply:
I thought I saw a comment or something that you wanted to move away from that construction so I've been trying to fix all those in the templates.  I don't know where I saw that.

Reply:
@bpalme, we moved away from the short tags `<? ... ?>` because it is not recommended anymore (we're using the less ambiguous `<?php ... ?>` instead). However, the short echo tag (`<?= ... ?>`) is still accepted, since it doesn't conflict with tags used by other languages. Sorry for the confusion!